db/schema.rb

```diff
@@ -184,10 +184,11 @@
 
   create_table "provider_users", force: :cascade do |t|
     t.string "email_address", null: false
-    t.string "dfe_sign_in_uid", null: false
+    t.string "dfe_sign_in_uid"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
-    t.index ["dfe_sign_in_uid"], name: "index_provider_users_on_dfe_sign_in_uid", unique: true
+    t.index ["dfe_sign_in_uid"], name: "index_provider_users_on_dfe_sign_in_uid"
```

Comment:
Sad to lose the index. Ought we to constrain DSI UIDs to be unique in the model now?